.vscode/settings.json

```diff
@@ -1,6 +0,0 @@
-{
-    "nxunitExplorer.nunit": "packages/NUnit.ConsoleRunner.3.9.0/tools/nunit3-console.exe",
-    "nxunitExplorer.modules": [
-        "bin/TestDebug/Xamarin.Android.Tools.AndroidSdk-Tests.dll"
-    ]
```

Comment:
Hmm yeah instead of removing this file maybe I could follow the advice here: https://stackoverflow.com/a/45650766

And set:
```json
    "dotnet-test-explorer.testProjectPath": "tests/Xamarin.Android.Tools.AndroidSdk-Tests/"

```

Reply:
so the nunit extension in vscode just runs the `nunit3-console.exe`, and nunit can not load .net core based assemblies so it has to be a net472* full framework type. 

If that `.NET Core test explorer" works then we should add that to the extensions list so that it gets installed automatically. We then might be able to use the setting above to replace the nunit stuff.

Reply:
This one might be better https://marketplace.visualstudio.com/items?itemName=derivitec-ltd.vscode-dotnet-adapter since it uses the https://marketplace.visualstudio.com/items?itemName=hbenl.vscode-test-explorer extension like our current ones do.

Reply:
Those two extensions seem to work well, I added them to the recommended list and updated the .settings file accordingly.